media/transcoder/create_job_with_periodic_images_spritesheet.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Transcoder sample for creating a job that generates two spritesheets from the input video. Each spritesheet contains images that are captured periodically.
+
+Example usage:
+    python create_job_with_periodic_images_spritesheet.py --project-id <project-id> --location <location> --input-uri <uri> --output-uri <uri>
+"""
+
+import argparse
+
+from google.cloud.video import transcoder_v1beta1
+from google.cloud.video.transcoder_v1beta1.services.transcoder_service import (
+    TranscoderServiceClient,
+)
+from google.protobuf import duration_pb2 as duration
+
+# [START transcoder_create_job_with_periodic_images_spritesheet]
```

Comment:
Please include the imports in the region tag - [See "Region Tags" in the Samples Format Guide](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#region-tags)

Reply:
Done. I have a b/182777706 to fix the rest of the samples.